plugins/woocommerce/src/Blocks/BlockTypes/ProductCollection.php

```diff
@@ -524,7 +524,10 @@ public function build_frontend_query( $query, $block, $page ) {
 		// phpcs:ignore WordPress.DB.SlowDBQuery
 		$block_context_query['tax_query'] = ! empty( $query['tax_query'] ) ? $query['tax_query'] : array();
 
-		$is_exclude_applied_filters = ! ( $block->context['query']['inherit'] ?? false );
+		$inherit    = isset( $block->context['query'], $block->context['query']['inherit'] ) && $block->context['query']['inherit'];
+		$filterable = isset( $block->context['query'], $block->context['query']['filterable'] ) && $block->context['query']['filterable'];
```

Comment:
Nitpick: I think we can use `??` operator here. It will make the code more readable i.e.

```php
$inherit    = $block->context['query']['inherit'] ?? false;
$filterable = $block->context['query']['filterable'] ?? false;
```

What do you think?

Reply:
Good point, done!